definitions/procedures/prep_6_10_upgrade.rb

```diff
@@ -0,0 +1,16 @@
+class Procedures::Prep610Upgrade < ForemanMaintain::Procedure
+  metadata do
+    description 'Prepare for the Satellite 6.10 upgrade'
+
+    confine do
+      feature(:satellite) && feature(:satellite).current_minor_version == '6.9'
+    end
+  end
+
+  def run
+    FileUtils.chmod_R "g=rwX", "/var/lib/pulp/content"
```

Comment:
Each of these commands might take quite a while, it might be good to print out what they are doing prior to running them

Reply:
Ok, I've added the output.